wiki/Beatmap_Ranking_Procedure/en.md

```diff
@@ -0,0 +1,43 @@
+# Beatmap Ranking Procedure
+
+Beatmaps can be promoted to the community by following the beatmap ranking procedure and entering the Ranked beatmaps category.
+
+## Feedback
+
+A beatmap's creator can mark their submissions as `Work-in-progress` or `Pending`. Beatmaps in both of these categories are intended to receive feedback from the osu! community.
```

Comment:
```suggestion
A beatmap's creator can mark their submissions as `Work in Progress` or `Pending`. Beatmaps in both of these categories are intended to receive feedback from the osu! community.
```
Could use this for consistency with everything around the site and osu!client too.